spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -543,4 +588,150 @@ definitions:
             desc: |
               Pseudorange and carrier phase observation for a
               satellite being tracked.
+ 
+ - MSG_OBS_DEP_B:
+    id: 0x0043
+    short_desc: Deprecated
+    desc: |
+      This observation message has been deprecated in favor of 
+      observations that are more interoperable. This message
+      should be used for observations referenced to 
+      a nominal pseudorange which are not interoperable with
+      most 3rd party GNSS receievers or typical RTCMv3 
+      observations.
+    public: False
+    replaced_by:
+      - MSG_OBS
+    fields:
+        - header:
+            type: ObservationHeader
+            desc: Header of a GPS observation message
+        - obs:
+            type: array
+            fill: PackedObsContentDepB
+            map_by: sid
+            desc: |
+              Pseudorange and carrier phase observation for a
+              satellite being tracked.
 
+ - MSG_EPHEMERIS_DEP_C:
+    id: 0x0047
```

Comment:
It is only for hte ID, but I have had big problems not being able to mask of the comparatively large ephemeris message so there is value in it.